homeassistant/components/weatherflow/sensor.py

```diff
@@ -0,0 +1,417 @@
+        translation_key="air_density",
+        native_unit_of_measurement="µg/m³",
+        device_class=SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=5,
+    ),
+)
+
+SENSORS: tuple[WeatherFlowSensorEntityDescription, ...] = (
+    WeatherFlowSensorEntityDescription(
+        key="air_temperature",
+        translation_key="temperature",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="dew_point_temperature",
+        translation_key="dew_point",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="feels_like_temperature",
+        translation_key="feels_like",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="wet_bulb_temperature",
+        translation_key="wet_bulb_temperature",
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="battery",
+        translation_key="battery voltage",
+        native_unit_of_measurement=UnitOfElectricPotential.VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="illuminance",
+        translation_key="illuminance",
+        native_unit_of_measurement=LIGHT_LUX,
+        device_class=SensorDeviceClass.ILLUMINANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="lightning_strike_average_distance",
+        icon="mdi:lightning-bolt",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.DISTANCE,
+        native_unit_of_measurement=UnitOfLength.KILOMETERS,
+        suggested_display_precision=2,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="lightning_strike_count",
+        translation_key="lightning_count",
+        icon="mdi:lightning-bolt",
+        state_class=SensorStateClass.TOTAL,
+    ),
+    WeatherFlowSensorEntityDescription(
+        key="precipitation_type",
+        device_class=SensorDeviceClass.ENUM,
+        options=["NONE", "RAIN", "HAIL", "RAIN_HAIL", "UNKNOWN"],
```

Comment:
Can we make these options lowercase? Then you can add the values to the translations files. Also, this sensor doesnt have a translation key

Reply:
I believe they come out of the backing library:

![image](https://github.com/home-assistant/core/assets/6491743/84b82197-f6ab-47b5-9a23-5f695a6bfc4d)

So I'm not sure if we can do that easily..

Reply:
if device_class == enum: return value.lower() :)

Reply:
not the actual code but you get the idea

Reply:
![image](https://github.com/home-assistant/core/assets/6491743/f712c226-ada2-4ec4-a990-42e83f50d1a4)

So I'll toss a `.lower()` on here and i think that will cut it

Reply:
Shouldn't it be picking up the translations? Or is not using the state object (I tried to copy what the moon sensor was doing)

Reply:
OOPS ... forgot to compile translations! :)